catalog/transformer/catalog-transformer-streaming-api/src/main/java/org/codice/ddf/transformer/xml/streaming/SaxEventHandler.java

```diff
@@ -32,4 +34,10 @@
      */
     List<Attribute> getAttributes();
 
+    /**
+     * Get all the possible attribute types that can be returned by this factory's handler
+     *
+     * @return a set of attribute descriptors that can be returned by this handler
+     */
+    Set<AttributeDescriptor> getSupportedAttributeDescriptors();
```

Comment:
This is the interface change we discussed a little while ago.
It was suggested that if it wasn't marked as experimental, it could be modified if the package wasn't listed in the whitelist.